Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -4631,6 +4632,55 @@ def com_google_fonts_check_varfont_instance_names(ttFont):
     yield PASS, "Instance names are correct"
 
 
+@check(
+  id = 'com.google.fonts/check/varfont_duplicate_instance_names',
+  rationale = """
+    According to Google Font’s specs (as of 2020-06-26), only weight and italic tokens are allowed in instance names (= "ExtraBold Italic").
+
+    This variable font contains duplicate instance names, suggesting that instances are defined for more then one axis.
+
+    Instead, only a base set of fonts for the most general representation of the family can be defined through instances in the `fvar` table,
+    all other instances will have to be left to access through the `STAT` table.
+  """,
+  conditions = ['is_variable_font'],
+)
+def com_google_fonts_check_varfont_duplicate_instance_names(ttFont):
+  """Check variable font instances don't have duplicate names"""
+  # This check and the fontbakery.parse module used to be more complicated.
+  # On 2020-06-26, we decided to only allow Thin-Black + Italic instances.
+  # If we decide to add more particles to instance names, It's worthwhile
+  # revisiting our previous implementation which can be found in commits
+  # earlier than or equal to ca71d787eb2b8b5a9b111884080dde5d45f5579f
```

Comment:
please remove this entire comment as it is leftover from copying code from **com.google.fonts/check/varfont_instance_names**. It is not true that "this check used to be more complicated" because this check is being freshly added right now ;-)